zp-relayer/validation/tx/validateTx.ts

```diff
@@ -65,10 +65,9 @@ export function checkTxSpecificFields(txType: TxType, tokenAmount: BN, energyAmo
   return null
 }
 
-export function checkNativeAmount(nativeAmount: BN | null) {
+export function checkNativeAmount(nativeAmount: BN | null, withdrawalAmount: BN) {
   logger.debug(`Native amount: ${nativeAmount}`)
-  // Check native amount (relayer faucet)
-  if (nativeAmount && nativeAmount > config.maxFaucet) {
+  if (nativeAmount && (nativeAmount > config.maxNativeAmount || nativeAmount > withdrawalAmount)) {
```

Comment:
How does it work? Should it be `X.gt(Y)` instead of `X > Y`?

Reply:
I confirm that issue. In the currently deployed relayer the `nativeAmount` compared with `config.maxNativeAmount` unpredictably incorrect

Reply:
Fixed